editor/src/components/canvas/controls/flow-slider-control.tsx

```diff
@@ -179,20 +168,36 @@ const AnimatedReorderIndicator = React.memo((props: AnimatedReorderIndicatorProp
     }
   }, 'FlowSliderControl indicatorOffset')
 
+  // when reaching the end of the slider it will restart from the beginning, a second indicator is shown
+  if (indicatorOffset > siblings.length - 1) {
```

Comment:
techinally you could _always_ render the two indicators :D one would be hidden most of the time because of the overflow anyways